src/Pos/Worker.hs

```diff
@@ -84,16 +84,13 @@ allWorkers NodeResources {..} = mconcatPair
       -- FIXME: perhaps these shouldn't be considered workers, but rather
```

Comment:
Is this TODO now done? The `TODO: Instead of basing this on nrKademlia we should look at the topology` one. That's addressed in this patch now right?